test/recipes/25-test_verify.t

```diff
@@ -141,6 +141,8 @@ ok(!verify("ee-cert", "sslserver", [], [qw(ca-cert)], "-partial_chain"),
    "fail untrusted partial chain");
 ok(verify("ee-cert", "sslserver", [qw(ca-cert)], [], "-partial_chain"),
    "accept trusted partial chain");
+ok(!verify("ee-cert", "sslserver", [qw(ca-expired)], [], "-partial_chain"),
```

Comment:
Should there be also a test without -partial_chain?

Reply:
Hmm actually because the ca-expired is trusted intermediate certificate and not self-issued one wouldn't this test pass even without the patch? I am afraid we need expired root cert for the test.

Reply:
Very true.
Luckily, using `mkcert.sh` it was easy to generate a suitable expired root cert.
The `setup.sh` now includes
```
DAYS=-1 ./mkcert.sh genroot "Root CA" root-key root-expired
```
I've also added a further x509 verify test that is now really to the point.